store/src/pmmr.rs

```diff
@@ -193,7 +193,13 @@ impl<T: PMMRable> PMMRBackend<T> {
 				data_dir.join(PMMR_LEAF_FILE).to_str().unwrap(),
 				header.hash()
 			);
-			LeafSet::copy_snapshot(&leaf_set_path, &PathBuf::from(leaf_snapshot_path))?;
+			// Check for a ... (3 dot) ending version of the file - could probably be removed after mainnet
+			let compaitble_snapshot_path = PathBuf::from(leaf_snapshot_path.clone() + "...");
```

Comment:
Small typo here: compatible. To your defense, I have a hard time typing too...

Reply:
The pitfalls of relying on autocorrect too much